src/main/java/nl/tudelft/ewi/devhub/server/database/controllers/Issues.java

```diff
@@ -0,0 +1,29 @@
+package nl.tudelft.ewi.devhub.server.database.controllers;
+
+import java.util.List;
+
+import javax.persistence.EntityManager;
+
+import com.google.inject.Inject;
+import com.google.inject.persist.Transactional;
+
+import nl.tudelft.ewi.devhub.server.database.entities.RepositoryEntity;
+import nl.tudelft.ewi.devhub.server.database.entities.issues.Issue;
+
+import static nl.tudelft.ewi.devhub.server.database.entities.issues.QIssue.issue;
+
+public class Issues extends Controller<Issue> {
+
+	@Inject
+	public Issues(EntityManager em) {
+		super(em);
+	}
+	
+	@Transactional
+	public List<Issue> findIssues(final RepositoryEntity repo){
+		return query().from(issue)
```

Comment:
This can potentially be replaced with:
```java
@OneToMany(mappedBy = "repository")
List<Issue> issues;
```
on `AbstractRepository`.

But maybe we will never really access all issues at once, and should we have separate queries for open and closed issues.